fda-model/structures/Inventory.cs

```diff
@@ -181,7 +193,7 @@ private void createColumnHeadersForMissingColumns(ref PointFeatureLayer layer, S
         {
             List<string> layerColumnNames = new List<string>();
             var row = layer.FeatureRow(0);
-            foreach (DataColumn c in row.Table.Columns)  //loop through the columns. 
+            foreach (System.Data.DataColumn c in row.Table.Columns)  //loop through the columns. 
```

Comment:
I have been told that best coding practices are to put comments above the line and not at the end of the line.

Reply:
I believe this method has been removed